deploy/homeserver/.env.caddy.prod.example

```
@@ -2,6 +2,11 @@
 # Key names only — never commit real secret values.
 API_DOMAIN=api.example.com
 WEB_DOMAIN=www.example.com
+# Company/product surfaces and the apex redirect (#1605). Optional: while unset each vhost stays on
+# its unreachable `.localhost` default. Never emit an empty value - see materialize_service_env.sh.
+COMPANY_DOMAIN=company.example.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COMPANY_DOMAIN key should go before the WEB_DOMAIN key

(UnorderedKey)